src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig

```diff
@@ -110,6 +110,32 @@
   col-sm input-container
 {%- endblock form_group_class %}
 
-{% block form_row_class -%}
-  form-group row{% if row_attr.class is defined %} {{ row_attr.class }}{% endif %}
-{%- endblock form_row_class %}
+{% block form_row_attr -%}
+  {% set classes = 'form-group row' %}
+  {% if (not compound or force_error|default(false)) and not valid %}
+    {% set classes = classes ~ ' has-error' %}
+  {% endif %}
+
+  {% if attr.visible is defined and not attr.visible %}
+    {% set classes = classes ~ ' d-none' %}
+  {% endif %}
+
+  {% if row_attr.class is defined %}
+    {% set classes = classes ~ ' ' ~ row_attr.class  %}
+  {% endif %}
+
+  {% set row_attr = row_attr|merge({'class': classes}) %}
+
+  {% for key, rowAttr in row_attr %} {{ key }}="{{ rowAttr }}" {% endfor %}
```

Comment:
That's a super feature, I also realized recently that `row_attr` could only be used for the `class` option
Now we're talking!